github/pulls_test.go

```diff
@@ -805,3 +805,32 @@ func TestPullRequestsService_Merge_options(t *testing.T) {
 		}
 	}
 }
+
+func TestPullRequestsService_Merge_Blank_Message(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+	madeRequest := false
+	expectedBody := ""
+	mux.HandleFunc("/repos/o/r/pulls/1/merge", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "PUT")
+		testBody(t, r, expectedBody+"\n")
+		madeRequest = true
+	})
+
+	ctx := context.Background()
+	expectedBody = `{"commit_message":""}`
```

Comment:
According to #1815, this will have no commit message, which is the case that I thought you added the `DontDefaultIfBlank` flag for.

Reply:
doh, I inverted the boolean 🤦 fixed this in the function body, which I felt was preferable to creating a duplicate struct